webapp/src/main/java/org/apache/atlas/web/rest/AuthREST.java

```diff
@@ -152,12 +155,26 @@ public ServicePolicies downloadPolicies(@PathParam("serviceName") final String s
                 perf = AtlasPerfTracer.getPerfTracer(PERF_LOG, "AuthREST.downloadPolicies(serviceName="+serviceName+", pluginId="+pluginId+", lastUpdatedTime="+lastUpdatedTime+")");
             }
 
-            if (!isPolicyUpdated(serviceName, lastUpdatedTime)) {
-                return null;
+            ServicePolicies ret;
+            if (usePolicyDelta) {
+                List<EntityAuditEventV2> auditEvents = getPolicyAuditLogs(serviceName, lastUpdatedTime);
+                long lastEventTime = auditEvents.isEmpty() ? 0 : auditEvents.get(auditEvents.size() - 1).getCreated();
+                LOG.info("PolicyDelta: serviceName={}, lastUpdatedTime={}, audit events found={}", serviceName, lastEventTime, auditEvents.size());
+                if (auditEvents.isEmpty()) {
+                    return null;
+                }
+                Map<String, EntityAuditEventV2.EntityAuditActionV2> policyChanges = policyTransformer.createPolicyChangeMap(serviceName, auditEvents);
+                ret = policyTransformer.getPoliciesDelta(serviceName, policyChanges, lastEventTime);
+            } else {
+                if (!isPolicyUpdated(serviceName, lastUpdatedTime)) {
+                    return null;
+                }
+                ret = policyTransformer.getPoliciesAll(serviceName, pluginId, lastUpdatedTime);
             }
-
-            ServicePolicies ret = policyTransformer.getPolicies(serviceName, pluginId, lastUpdatedTime);
-
+            LOG.info("downloadPolicies: serviceName={}, lastUpdatedTime={}, policies fetched={} delta fetched={}", serviceName,
```

Comment:
is this info needed given that we already log policy count in auth-agents-common/src/main/java/org/apache/atlas/policytransformer/CachePolicyTransformerImpl.java